content/1.3/docs/core-concepts/workflow-types/_index.md

```diff
@@ -65,7 +65,7 @@ The *workflowOptions* object must possess six essential attributes with specific
 ```
 
 ##### Examples:
-- [MTA](https://github.com/parodos-dev/serverless-workflows/blob/v1.3.x/mta/mta.sw.yaml)
+- [MTA](https://github.com/parodos-dev/serverless-workflows/blob/v1.3.x/mta-v7.x/mta.sw.yaml)
```

Comment:
Maybe https://github.com/parodos-dev/serverless-workflows/commit/b91d50d7f85c5abe1858d573031b83c5a4ff2130 needs to be backported to 1.3 too?

Reply:
probably, since we wouldn't like users to use the unsupported mta version. can you raise a PR for it?